ci/container_index/run.py

```diff
@@ -0,0 +1,125 @@
+import sys
+from os import path
+
+from argparse import ArgumentParser
+import ci.container_index.engine as engine
+
+
+def init_parser():
+    parser = ArgumentParser(
+        description="This tool lets you run index validations from the CLI"
+    )
+    parser.add_argument(
+        "-i",
+        "--index",
+        help="Specify location of container index. This is where index.d is. "
+             "'./'is default",
+        metavar="INDEX_PATH",
+        default="."
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        help="Verbose mode prints more output of the tests.",
+        action="store_true"
+    )
+    parser.add_argument(
+        "-s",
+        "--schemavalidators",
+        help="Comma seperated list of schema validators to run."
+             " to skip all",
```

Comment:
I think the help text needs an update?

` to skip all` ?

Reply:
Sorted